src/print.c

```diff
@@ -53,6 +54,20 @@ int apk_get_screen_width(void)
 	return apk_screen_width;
 }
 
+char apk_get_human_unit(size_t size, double *dest)
+{
+	char units[] = {'B', 'K', 'M', 'G', 'T'};
+	size_t i;
+	double s;
+
+	for (i = 0, s = size; s >= 1024 && i < sizeof(units); i++)
```

Comment:
Prefer to use `ARRAY_SIZE()` instead of `sizeof` here. Yes the end result is same as general sizeof(char)==1, but the intention is more clear.